packages/components/Avatar/src/AvatarTokens.android.ts

```diff
@@ -1,171 +1,3 @@
-import type { Theme } from '@fluentui-react-native/framework';
-import type { TokenSettings } from '@fluentui-react-native/use-styling';
-import type { AvatarTokens } from '.';
-import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import { defaultAvatarTokens } from './AvatarTokens.mobile';
```

Comment:
Does just `export { defaultAvatarTokens } from './AvatarTokens.mobile' work

Reply:
Somehow I get the error `@rnx-kit/no-const-enum` when I do so. I'll take a deeper look later.

Reply:
Ah okay interesting. I think for compoundbutton I was able to just export it in one line

Reply:
I take it back, I think it works!